dev_tools/scripts/scw

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+if `command -v sem`.empty?
```

Comment:
```suggestion
if `which -s sem`
```

Reply:
oh right, it should be `if system('which', '-s', 'sem')`

System translates the exit status into true/false.

Reply:
or, I guess `if system('command', '-v', 'sem')` given the advice to avoid `which` :)

Reply:
It would have to be `unless` instead of `if`.

I'll just stick with the `empty?` check with backticks.